src/AddIns/Uno.UI.MSAL.Wasm/WasmScripts/uno-ui-msal.d.ts

```diff
@@ -0,0 +1,6 @@
+declare namespace MSAL {
+    class WebUI {
+        static authenticate(urlNavigate: string, urlRedirect: string, title: string, popUpWidth: number, popUpHeight: number): Promise<string>;
```

Comment:
Codacy found an issue: [Exceeds maximum line length of 120](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5872302)